README.md

```diff
@@ -37,7 +37,7 @@ Using [`vim-plug`](https://github.com/junegunn/vim-plug):
 ```vim
 Plug 'autozimu/LanguageClient-neovim', {
     \ 'branch': 'next',
-    \ 'do': 'bash install.sh',
+    \ 'do': 'bash install.sh && :UpdateRemotePlugins',
```

Comment:
This is actually no longer required. This plugin was used to implemented as neovim remote plugin, but that's no longer true after a rewrite to support vim as well.